doc/usr/source/9_other/5_proofs.rst

```diff
@@ -78,41 +78,39 @@ Requirements
 ^^^^^^^^^^^^
 
 Frontend certificates and proofs production require the user to have JKind
-installed on his machine (together with a suitable version of Java).
+installed on their machine (together with a suitable version of Java).
 
 SMT-LIB 2 certificates do not require anything additional excepted for an SMT
 solver to check the certificates.
 
-LFSC proofs production require a proof producing version of cvc5 (the binary
-can be specified with ``--cvc5_bin``), and the LFSC checker to be compiled for
-the final proof checking phase.
+LFSC proofs production requires cvc5 (the binary can be specified with
+``--cvc5_bin``), its LFSC proof signatures, and the LFSC checker for the final
+proof checking phase.
 
 LFSC checker
 ~~~~~~~~~~~~
 
-The LFSC checker is also distributed with Kind 2 in the directory ``lfsc``, it
-contains the checker and the necessary signature files with the proof rules:
+A bash script to download and build the LFSC checker is distributed with Kind 2.
+The script also generates an easy-to-use bash script to check LFSC proofs
+generated by Kind 2:
 
 .. code-block:: none
 
    lfsc
-   |-- checker
-   |   ...
+   |-- get-lfsc-checker.sh
+   |-- bin
+       |-- lfsc-check.sh
+       |-- lfscc
+   |   ...
    |-- signatures
        |-- kind.plf
-       |-- sat.plf
-       |-- smt.plf
-       |-- th_base.plf
-       |-- th_int.plf
-       |-- th_real.plf
+       |-- ...
 
-The checker can be compiled using:
+The checker can be downloaded and compiled using:
 
 .. code-block:: none
 
-   autoreconf -i
-   ./configure 
-   make
+   contrib/get-lfsc-checker.sh
```

Comment:
`contrib/get-lfsc-checker.sh` should be `./lfsc/get-lfsc-checker.sh`

Reply:
I prefer to avoid the dot (it looks ugly to me...)

Reply:
Are you sure? `lfsc/get-lfsc-checker.sh` works just fine for me...

Reply:
You are right. When you specify a directory, the dot is not required. Fine then.